tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
This is a good point, but the existence of `configure` is to allow creating a default settings instance (at config/__init__.py) and then avoid doing the parsing at import time. Maybe we can try to do a lazy evaluation here but we will need to alert clients to just use settings at runtime, otherwise will break documentation.